projects/1.19/assets/dungeons-mobs/dungeons_mobs/lang/zh_cn.json

```
@@ -0,0 +1,524 @@
+  
+  "item.dungeons_mobs.drowned_necromancer_helmet": "溺尸死灵法师头冠",
+  "item.dungeons_mobs.drowned_necromancer_chestplate": "溺尸死灵法师斗篷",
+  "item.dungeons_mobs.drowned_necromancer_leggings": "溺尸死灵法师腰带",
+  "lore.dungeons_mobs.drowned_necromancer_armor": "沉默在风暴中的溺尸服饰，沉没许久的遗物，据传死灵法师仍然崇拜来自深渊的独眼神。",
+
+  "item.dungeons_mobs.enchanter_hat": "附魔师帽",
+  "item.dungeons_mobs.enchanter_robes": "附魔师长袍",
+  "lore.dungeons_mobs.enchanter_clothes": "只有灾厄帝国中的附魔师才有资格穿戴镶有钻石的猩红色长袍",
+  
+  "item.dungeons_mobs.windcaller_helmet": "唤风者假发",
+  "item.dungeons_mobs.windcaller_chestplate": "唤风者长袍",
+  "lore.dungeons_mobs.windcaller_armor": "来自嚎风山巅的唤风者穿戴过的装备，唤风者那高大的假发是他们与风交流时必不可少的道具。",
+  
+  "item.dungeons_mobs.fungus_thrower_helmet": "猪灵掷菌手头巾",
+  "item.dungeons_mobs.fungus_thrower_chestplate": "猪灵掷菌手背包",
+  "lore.dungeons_mobs.fungus_thrower_armor": "在猪灵家族的文化中，紫色被认为是最狡猾的颜色之一，只有精心制作的麻袋才能装下来自诡异森林中的致命真菌。",
+  
+  "item.dungeons_mobs.netherplate_helmet": "下界板头盔",
+  "lore.dungeons_mobs.netherplate_armor": "野火的黑色网管，下界堡垒的守护者，据说野火的这个深红色宝石包含了下界的灼热能量。",
+
+  "item.dungeons_mobs.vanguard_shield": "先锋盾牌",
+  "lore.dungeons_mobs.vanguard_shield": "经过漫长岁月的洗礼并没有使这面盾牌变得脆弱，据传是无名国王中的先锋护卫队使用过的一种无法破坏盾牌。",
+
+  "item.dungeons_mobs.royal_guard_shield": "皇家守卫盾牌",
+  "lore.dungeons_mobs.royal_guard_shield": "在这面盾牌上印有来自教会的黄色十字架，代表着皇家卫队的神圣职责的同时也代表着教会的强大。",
+  "item.dungeons_mobs.royal_guard_shield.black": "黑色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.red": "红色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.green": "绿色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.brown": "棕色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.blue": "蓝色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.purple": "紫色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.cyan": "青色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.light_gray": "淡灰色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.gray": "灰色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.pink": "粉红色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.lime": "黄绿色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.yellow": "黄色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.light_blue": "淡蓝色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.magenta": "品红色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.orange": "橙色皇家守卫盾牌",
+  "item.dungeons_mobs.royal_guard_shield.white": "白色皇家守卫盾牌",
+
+  "itemGroup.dungeonsMobs": "地下城：生物",
+  "itemGroup.dungeonsMobsItems": "地下城：生物装备",
+
+  "mobEnchantment.dungeons_mobs.burning": "灼烧",
+  "mobEnchantment.dungeons_mobs.chilling": "冻结",
+  "mobEnchantment.dungeons_mobs.deflect": "偏转",
+  "mobEnchantment.dungeons_mobs.double_damage": "双倍伤害",
+  "mobEnchantment.dungeons_mobs.echo": "回击",
+  "mobEnchantment.dungeons_mobs.fire_trail": "焰尾",
+  "mobEnchantment.dungeons_mobs.gravity_pulse": "引力脉冲",
+  "mobEnchantment.dungeons_mobs.levitation_shot": "悬浮射击",
+  "mobEnchantment.dungeons_mobs.multishot": "多重",
+  "mobEnchantment.dungeons_mobs.protection": "保护",
+  "mobEnchantment.dungeons_mobs.quick": "快速",

```

```suggestion
  "mobEnchantment.dungeons_mobs.quick": "疾速",
```
可以参考[wiki的相关资料](https://minecraft.fandom.com/zh/wiki/Minecraft_Dungeons:%E9%99%84%E9%AD%94)